src/headless/converse-roster.js

```diff
@@ -655,10 +655,9 @@ converse.plugins.add('converse-roster', {
              */
             updateContact (item) {
                 const jid = item.getAttribute('jid');
-                if (this.isSelf(jid)) { return; }
 
                 const contact = this.get(jid);
-                const subscription = item.getAttribute("subscription");
+                const subscription = this.isSelf(jid) ? 'both' : item.getAttribute("subscription");
```

Comment:
This feels too hacky. What happens when you just remove line 658, without making this change?

Reply:
Because of `subscription === null` the roster item won't be shown.
I updated the PR to make it less hacky: Now the case of a roster item belonging to yourself is handled in rosterview.

Reply:
Looks better thanks.